config.yaml

```
@@ -0,0 +1,15 @@
+DatabaseConfig:
+  DbName: "postgres"
+  User: "postgres"
+  Password: "secret"
+  Address: "postgres:5432"
+
+ServerConfig:
+  Port: ":8080"
+  ShutdownTimeout: 3s
+  Module : "server"
```

```suggestion
server:
  port: ":8080"
  shutdown_timeout: 3s
```
Что-то не понял, зачем тут "module" - у нас нет необходимости конфигурировать т.е. изменять это значение. При этом обрати внимание, что мы уже находимся в файле `config.yaml`, поэтому нет необходимости повторяться и говорить, что это `server_config`